requirements/edx/base.txt

```diff
@@ -48,7 +48,7 @@ edx-lint==0.4.3
 astroid==1.3.8
 edx-django-oauth2-provider==1.1.4
 edx-django-sites-extensions==2.3.0
-edx-enterprise==0.39.9
+edx-enterprise==0.40.0
```

Comment:
Maybe we should update this to 0.40.1 since that is the latest version of enterprise?